hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java

```diff
@@ -59,6 +62,59 @@ public void testEnsureFileCreatedImmediately() throws Exception {
     assertIsFile(fs, TEST_FILE_PATH);
   }
 
+  /**
+   * {@link AbfsOutputStream#incrementWriteOps()}
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testWriteOpsMetric() throws Exception {
+    describe("Test to see correct population of write operations in Abfs");
+    final AzureBlobFileSystem fs = getFileSystem();
+    Path smallFile = new Path("testOneCall");
+    Path largeFile = new Path("testLargeCalls");
+    String testWriteOps = "test";
+    FileSystem.Statistics statistics = fs.getFsStatistics();
+    statistics.reset();
+
+    //Test for zero write operation
+    Assert.assertEquals(0, statistics.getWriteOps());
+
+    FSDataOutputStream outForOneCall = fs.create(smallFile);
+    statistics.reset();
+    //Writing "test" 1 time
+    outForOneCall
+        .write(testWriteOps.getBytes(), 0, testWriteOps.getBytes().length);
+
+    //Test for one write operation
+    Assert.assertEquals(1, statistics.getWriteOps());
```

Comment:
All assert calls should have meaningful message as suggested by Steve. This is important for debugging test failures easily. You can check the examples in PR I sent you. 
Sorry I missed this in internal review. :)

Reply:
Noted.